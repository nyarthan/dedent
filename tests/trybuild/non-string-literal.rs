use dedent::dedent;

fn main() {
    let str = dedent!(1);
}
