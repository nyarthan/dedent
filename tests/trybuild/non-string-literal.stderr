error: expected string literal
 --> tests/trybuild/non-string-literal.rs:4:23
  |
4 |     let str = dedent!(1);
  |                       ^
